{
  "comments": [
    {
      "key": {
        "uuid": "ba1f61a0_11456042",
        "filename": "csit/suites/openstack/connectivity/03_external_network_tests.robot",
        "patchSetId": 3
      },
      "lineNbr": 83,
      "author": {
        "id": 2264
      },
      "writtenOn": "2017-09-28T18:47:43Z",
      "side": 1,
      "message": "shouldn\u0027t we add some checks after this to ensure the\nbug is not there? I\u0027m guessing this Update Network command\nis going to work. Per the bug, we should check the odl-nat:external-networks model and verify a flow is there in\nthe FIB table (21)",
      "range": {
        "startLine": 83,
        "startChar": 4,
        "endLine": 83,
        "endChar": 56
      },
      "revId": "a146172ea8895f1155067fc884173f3346a1715e",
      "serverId": "7fc14799-209e-464c-9743-7a06c2c21a81",
      "unresolved": false
    }
  ]
}